.buildkite/steps/prepare-release.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env sh
+
+current=$(svu current)
+
+
+options() {
+  if [[ "$current" =~ "-rc" ]]; then
+    # if current version is 0.7.5-rc.1
+    # the options should be
+    # - 0.7.5
+    # - 0.7.5-rc.2
+    cat <<YAML
+        - label: $(svu patch)
+          value: $(svu patch)
+        - label: $(svu prerelease)
+          value: $(svu prerelease)
+YAML
+  else
+    # if current version is 0.7.5
```

Comment:
We currently don't need to worry about releasing major version right? We could add that before releasing the first major version

Reply:
Yes, I think the plan is to have a mechanism to override that options, maybe type the version number manually on special circumstance.